src/pages/Home/About.js

```diff
@@ -25,42 +25,67 @@ const AboutText = styled.div`
   line-height: 1.5em;
   font-weight: 300;
   text-align: center;
+  text-indent: 1.5rem;
   color: #2e2e2e;
   margin: 0 auto;
+  width: 48ch;
```

Comment:
It looks like this change is preventing the copy from being centered at the 375 breakpoint. Check the red and green lines in the screenshot:
![Screen Shot 2022-05-15 at 2 44 40 PM](https://user-images.githubusercontent.com/49624436/168495251-fba186cc-eb3a-4ac1-99d6-0d7014aff11a.png)

Reply:
I *think* simply removing the `width: 48ch` line fixes things, but maybe I'm then again maybe I'm overlooking the original issue you were trying to fix with adding the `width: 48ch` . . .